src/main/java/sg/ncl/domain/ExceptionState.java

```diff
@@ -21,5 +21,6 @@
     CredentialsNotFoundException,
     EmailNotVerifiedException,
     UserNotApprovedException,
-    InvalidCredentialsException
+    InvalidCredentialsException,
+    EmailAlreadyExistsException
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00115)